cli/test/test_cmdversion.py

```diff
@@ -0,0 +1,21 @@
+from click.testing import CliRunner
+import popper.commands.cmd_version as version
+import unittest
+from popper_test import PopperTest
+from popper import __version__ as version_value
+
+
+class TestCommandVersion(unittest.TestCase, PopperTest):
+
+    def test_version(self):
+
+        with self.assertLogs('popper') as test:
+            runner = CliRunner()
+            result = runner.invoke(version.cli)
+            assert result.exit_code == 0
```

Comment:
this and the following two assertions can use `unittest` assertions instead, like we do in other tests